package.json

```diff
@@ -1,20 +1,21 @@
 {
   "name": "tic-tac-toe-backend",
   "version": "1.0.0",
+  "type": "module",
   "dependencies": {
-    "@aws-sdk/client-dynamodb": "^3.145.0",
-    "aws-sdk": "^2.1193.0",
-    "connect-dynamodb": "^2.0.5",
+    "@aws-sdk/client-dynamodb": "^3.150.0",
+    "aws-sdk": "^2.1199.0",
     "express": "^4.18.1"
   },
   "devDependencies": {
     "eslint": "^8.21.0",
     "prettier": "2.7.1"
   },
-  "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit",
+    "db-init": "node index.js"
```

Comment:
Usually the name `index.js` is used as an entry point for our service. So it's a good idea to name it something else.

And also. Starting the setup script in the following manner would not work, because we have not set the **SETUP_DB** env variable to anything. 
Hint 👇
https://www.twilio.com/blog/2022/02/working-with-environment-variables-in-node-js.html